packages/block-library/src/heading/edit.native.js

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
Hm, why was this duplicated?

Reply:
Native code doesn't support `useBlockEditingMode`